### ISSUE TITLE
Banner component makes it possible to have multiple h1 on a page.

### ISSUE THREAD

**Issue by amazingphilippe** [cds-snc/notification-planning#816]
## Description of issue

Describe what is failing
Banners are configured to use a h1. Sometimes, that h1 may be the only one on a page, sometimes, we may have another h1 on a page already.

Either way, having the h1 in a banner can be weird when the message is quite long. It also doesn't look like a heading for the whole page, and often mismatched with the page title attribute. 

There can only be one h1 per page. See [this reference](https://www.a11yproject.com/posts/how-to-accessible-heading-structure/) for a very good breakdown. [Including some recently resolved confusion](https://www.a11yproject.com/posts/how-to-accessible-heading-structure/#one-h1)

![Capture d’écran, le 2022-08-17 à 15.50.25.png](https://images.zenhubusercontent.com/614b439e980ccd4137400d3e/e5cddd38-7222-443f-8d64-18a1b06d0497)

Also some banners are nested in another banner. This would be a good opportunity to fix. 

![Capture d’écran, le 2022-08-17 à 15.49.51.png](https://images.zenhubusercontent.com/614b439e980ccd4137400d3e/c126158c-06d1-4171-bddd-17b2dea5c77a)

## Potential fix

- Change the banner component to default to a h2
- Make sure pages where we are using the component have a regular h1. 
- - Put each page in a quantitative table
- - Design headings for those pages, or use the page title prop if it makes sense
- Un-nest nested banners

## Resources

- Links to documentation or articles that could be helpful
- examples: WCAG docs, ARIA docis, web.dev, Deque, Webaim...

**Comment by amazingphilippe** [comment]
https://www.w3.org/WAI/WCAG21/quickref/?showtechniques=246#section-headings

**Comment by YedidaZalik** [comment]
Adding to Ally stmt

**Comment by andrewleith** [comment]
Notes:
- The `banner()` macro draws an `h1` when the parameter `subhead` is passed with a string of text.
- `banner()` with a `subhead` param is only called via `banner_wrapper()` from `template.html` when `show_redaction_message=True` - which should be the only case we are interested in for this card
- The screenshots shown in this card are actually hard-coded as h2's, and do not use the `banner` macro

The only place I can find where this would actually get used, is when a user clicks on 'Redact personalised variable content after sending' on the template screen.  In this case, we do end up with 2 h1s on the page.

**Comment by YedidaZalik** [comment]
Andrew has moved to review and has tagged folks
